### ISSUE TITLE
Occlusion with Set Teeth on Curve

### ISSUE THREAD

**Issue by patmo141** [patmo141/odc_public#40]
The operator Set Teeth on Curve currently re-scales the upper and lower arches to be exactly the same length, which in clinical reality is not the case.  To better be able to simulate real/harmonious occlusal schemes, a smarter calculation will be better.

TOP:  DESIRED BEHAVIOR
BOTTOM:  CURRENT BEHAVIOR

![image](https://cloud.githubusercontent.com/assets/2917743/22150943/d8a5df0a-dee9-11e6-8335-5e4a313e4af4.png)

**Comment by Guisan** [comment]
Very good!
Is it also possible to tilt the incisal zone of the canines and the incisors toward the vestibule?
![screenshot 70](https://cloud.githubusercontent.com/assets/17667365/22167628/ed4f618c-df67-11e6-9bb8-49ca719073cc.jpg)

**Comment by patmo141** [comment]
yes but that will have to come from the design of the tooth library.  I will post more information on Wiki once I have it all sorted out.  Marcos Graf has been working on adjusting the tooth library so that the tip and torque are stored in the local coordinates of the tooth.

**Comment by patmo141** [comment]
![image](https://cloud.githubusercontent.com/assets/2917743/22276052/1a992d98-e27f-11e6-9d8d-21b4215faa75.png)

One problem is that the bounding box width does not represent the width of the tooth along the curve, especially on the lingual contacts of the upper anteriors.

**Comment by patmo141** [comment]
![image](https://cloud.githubusercontent.com/assets/2917743/22276211/1f6f162e-e280-11e6-847b-8dd568b0e1e0.png)

This method works better by taking a cross section at the occlusal point and using that to measure

**Comment by patmo141** [comment]
![test_1](https://cloud.githubusercontent.com/assets/2917743/22305578/0acdb628-e30a-11e6-9f8e-3a051cf64263.gif)
